src/Shared/Neptune/Messages/CatalogMessages.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Neptune.Services;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Neptune.Messages.Catalog
+{
+    class CatalogHealth : GetRequest<ICatalogApplication>
```

Comment:
Get requests do not support having a request body, therefore if the uri has uri parameters, then the message request will have to build it.

@isaac-gs Do you have any suggestions?

Reply:
There is no way we can really make that type safe. 

You can have Uris that look like api/users/:I'd/post/:eventid?amount=3 and there's no way we can really do anything about it